lib/pages/contacts/contacts.dart

```diff
@@ -0,0 +1,117 @@
+import 'package:after_layout/after_layout.dart';
+import 'package:fluffychat/pages/contacts/contacts_view.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_contacts/contact.dart';
+import 'package:flutter_contacts/flutter_contacts.dart';
+import 'package:permission_handler/permission_handler.dart';
+
+class Contacts extends StatefulWidget {
+  const Contacts({Key? key}) : super(key: key);
+
+  @override
+  State<Contacts> createState() => ContactsController();
+}
+
+class ContactsController extends State<Contacts> with AfterLayoutMixin<Contacts> {
```

Comment:
What is `AfterLayoutMixin`?

Reply:
It allow schedule a run after first build of widget